src/library_html5.js

```diff
@@ -2656,6 +2656,7 @@ var LibraryJSEvents = {
     cancelAnimationFrame(id);
   },
 
+  emscripten_request_animation_frame_loop__deps: ['$getDynCaller'],
```

Comment:
This kind of deps mechanism is a bit awkward since it leaks the need to know internal names from `makeDynCall` out to the JS code. Can we align the naming so that one would type the same string into the __deps field that one puts inside the {{{ }}} block. That way there is at least a visual hint to suggest where the dependency comes from. (otherwise it can be hard to discover)

Reply:
I manged to remove the need for these.